docs/extending/adding_reports.md

```diff
@@ -178,6 +178,31 @@ def register_unpublished_changes_report_url():
 
 Here, we use the `AdminOnlyMenuItem` class to ensure our report icon is only shown to superusers. To make the report visible to all users, you could replace this with `MenuItem`.
 
+## Setting up permission restriction
+
+To make sure that the reports are only accessible by the `Admin` it is required to set up a permission restriction on the report view itself this can be done by adding a `dispatch` method to the existing `UnpublishedChangesReportView` view .
```

Comment:
We've referred "superusers" just above, rather than "the admin", and I think it's better to be consistent - "Admin" is a bit ambiguous, and can refer to the Wagtail admin interface in general. (In retrospect, `AdminOnlyMenuItem` was badly named...) Also, "Admin" doesn't need to be formatted as code here.

I think it's also worth giving a bit more explanation about why this is different from setting a permission on the menu item:

```suggestion
Even with the menu item hidden, it would still be possible for any user to visit the report's URL directly, and so it is necessary to set up a permission restriction on the report view itself. This can be done by adding a `dispatch` method to the existing `UnpublishedChangesReportView` view:
```